src/test/java/com/sap/oss/phosphor/fosstars/data/github/SecurityReviewsFromOpenSSFTest.java

```diff
@@ -151,17 +162,53 @@ public void testReadReviewDateFrom() throws JsonProcessingException {
   }
 
   @Test
-  public void testFetchValueFor() throws IOException {
-    Path repositoryDirectory
+  public void testFetchValueFor() throws IOException, GitAPIException, ParseException {
+    Path securityReviewDirectory
         = Files.createTempDirectory(SecurityReviewsFromOpenSSFTest.class.getName());
-    try {
-      LocalRepository repository = mock(LocalRepository.class);
-      TestGitHubDataFetcher.addForTesting(SECURITY_REVIEWS_PROJECT, repository);
+    Path directory = Files.createTempDirectory(getClass().getName());
+    try (Repository repository = FileRepositoryBuilder.create(directory.resolve(".git").toFile());
+        Git git = new Git(repository)) {
```

Comment:
Seems to be an indentation issue

Reply:
That was auto-formatted according to the formatting rules. I can add an extra space here.